lib/google_drive/config.rb

```diff
@@ -0,0 +1,36 @@
+require 'json'
```

Comment:
Can you put copyright line here like other files?:

    # Author: Hiroshi Ichikawa <http://gimite.net/>
    # The license of this source is "New BSD Licence"

Please use your name as Author instead.